virt-v2v/warm/.bazelrc

```diff
@@ -0,0 +1,6 @@
+# Appliance build
+# container_run_and_extract() does not work inside Podman and Docker
+# sandboxes. Use slightly less secure but working processwrapper sandbox.
+# NOTE: Same configuration is in .bazelrc in repository root.
+build --strategy_regexp="Action appliance/libguestfs-appliance.tar"=processwrapper-sandbox
```

Comment:
we also set it in the root `.bazelrc`. not sure if it's needed in both places.

Reply:
right, it's already there